NuKeeper/Configuration/CommandLineArguments.cs

```diff
@@ -8,7 +9,8 @@ public class CommandLineArguments
         [CommandLine("mode", "m"), Required]
         public string Mode;
 
-        [CommandLine("github_token", "t"), Required, SensitiveInformation]
+        [Environment("NuKeeper_github_token"), Required, SensitiveInformation]
+        [OverriddenBy(ConfigurationSources.CommandLine, "t")]
```

Comment:
maybe if we're overriding, we don't want it to be necessarily easy - so should have long names?